docs/en/cloud/security/azure-privatelink.md

```diff
@@ -153,31 +156,31 @@ resource "azurerm_private_endpoint" "example_clickhouse_cloud" {
 }
 ```
 
-### Obtaining Private Endpoint resourceGuid
+### Obtaining the Private Endpoint resourceGuid {#obtaining-private-endpoint-resourceguid}
 
-In order to use Private Link, you need to add Private Endpoint connection GUID to instance allow list.
+In order to use Private Link, you need to add the Private Endpoint connection GUID to your service allow list.
 
-#### Azure portal
-
-Private Endpoint resource GUID is only exposed in Azure portal. Open Private Endpoint created in previous step and click `JSON View`.
+The Private Endpoint resource GUID is only exposed in the Azure Portal. Open the Private Endpoint created in previous step and click **JSON View**:
 
 ![PE GUID](@site/docs/en/cloud/security/images/azure-pe-view.png)
 
-Under properties find `resourceGuid` field and copy this value:
+Under properties, find `resourceGuid` field and copy this value:
 
 ![PE GUID](@site/docs/en/cloud/security/images/azure-pe-resource-guid.png)
 
 ## Setting up DNS for Private Link
 
-You need will need to create Private DNS zone ${location_code}.privatelink.azure.clickhouse.cloud and attach it to VNET to access resources via Private Link.
+You need will need to create a Private DNS zone and attach it to your VNET to access resources via Private Link.
```

Comment:
It's not obvious that customer need to create ${region_code}.privatelink.azure.clickhouse.cloud zone. Let's add it somewhere ?

Reply:
Should I add back the following?

>You need will need to create Private DNS zone `${location_code}.privatelink.azure.clickhouse.cloud` and attach it to VNET to access resources via Private Link.

And is it `location_code` or `region_code` (you referenced both)? Sorry, I have not used Azure before so I am unfamiliar with what this URL would be used for. Maybe adding a short description would be helpful.

Reply:
In azure region called as "location", that's why I put `location_code` here. I am ok with both options, not sure what is less confusing thought.